Mixer.py

```diff
@@ -25,6 +28,15 @@ def mix(self, number_of_records=1000, with_incremental_id=False):
             result.append(line)
         return result
 
+    def json(self, number_of_records=30):
+        result = []
+        self.delimiter = ','
+        for id in range(number_of_records):
+            line = self.delimiter.join(random.choice(list) for list in self.columns)
+            line = '{'+line+'}'+'\n'
```

Comment:
new line in the end seems excessive. Let code that handles result of mix()/json() decide how to handle new lines